KarhooUISDK/Screens/QuoteList/QuoteListTable/QuoteListTablePresenter.swift

```diff
@@ -61,7 +61,7 @@ class KarhooQuoteListTablePresenter: QuoteListTablePresenter {
         case .empty(reason: .originAndDestinationAreTheSame):
             return UITexts.Errors.errorPickupAndDestinationSameTitle
         case .empty(reason: .noAvailabilityInRequestedArea):
-            return "No fleets in this area yet"
+            return UITexts.KarhooError.K3002
```

Comment:
Please don't forget to merge this in the release branch as well so that we don't get a regression next time around